creator-node/src/fileManager.js

```diff
@@ -1,17 +1,8 @@
 const path = require('path')
-const fs = require('fs')
+const fs = require('fs-extra')
 const multer = require('multer')
 const getUuid = require('uuid/v4')
 const axios = require('axios')
-const { promisify } = require('util')
-
-const lstat = promisify(fs.lstat)
```

Comment:
all replaced by `fs-extra` calls

Reply:
since these old fs were all awaited does this change makes any difference?

Reply:
agreed, it shouldn't - the main thing i wanted to change was the synchronous `fs.existsSync()`, i just changed these since i was making related changes